src/opentelemetry/lightstep/configuration.py

```diff
@@ -190,13 +212,12 @@ def configure_opentelemetry(
     if access_token != "":
         metadata = (("lightstep-access-token", access_token),)
 
+    credentials = None
+
     # FIXME rename insecure to secure in order to avoid a negation
-    if not insecure:
+    if not span_exporter_endpoint_insecure:
```

Comment:
this could be updated to something like the following, which would take care of the FIXME
```
if span_exporter_endpoint_insecure:
  credentials = None
else:
  credentials = ssl_channel_credentials()
```

Reply:
Fixed